doc/install/containers.rst

```diff
@@ -0,0 +1,86 @@
+Ceph Container Images
+=====================
+
+Official Releases
+-----------------
+
+Ceph Container images are available from Docker Hub at::
+
+  https://hub.docker.com/r/ceph
+
+
+ceph/ceph
+^^^^^^^^^
+
+- General purpose Ceph container with all necessary daemons and
+  dependencies installed.
+
++------------------+--------------------------------------------------------------+
+| Tag              | Meaning                                                      |
++------------------+--------------------------------------------------------------+
+| vRELNUM          | Latest release in this series (e.g., *v14* = Nautilus)       |
++------------------+--------------------------------------------------------------+
+| vRELNUM.2        | Latest *stable* release in this stable series (e.g., *v14.2*)|
++------------------+--------------------------------------------------------------+
+| vRELNUM.2.DOTNUM | A specific release (e.g., *v14.2.4*)                         |
++------------------+--------------------------------------------------------------+
+
+ceph/daemon-base
+^^^^^^^^^^^^^^^^
+
+- General purpose Ceph container with all necessary daemons and
+  dependencies installed.
+- Basically the same as *ceph/ceph*, but with different tags.
+
+:note: This image will soon become an alias to *ceph/ceph*.
+
++-----------------------+-----------------------------------------------------+
+| Tag                   | Meaning                                             |
++-----------------------+-----------------------------------------------------+
+| latest-master         | ???                                                 |
++-----------------------+-----------------------------------------------------+
+| latest-master-devel   | Daily build of the master branch                    |
++-----------------------+-----------------------------------------------------+
+| latest-RELEASE-devel  | Daily build of the *RELEASE* (e.g., nautilus)       |
```

Comment:
maybe mentioned that the `-devel` are based on the ceph unreleased packages (shaman/chachra)